platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/PlatformBuilder.java

```diff
@@ -0,0 +1,272 @@
+import static com.swirlds.platform.StaticPlatformBuilder.doStaticSetup;
+import static com.swirlds.platform.StaticPlatformBuilder.getGlobalMetrics;
+import static com.swirlds.platform.StaticPlatformBuilder.getMetricsProvider;
+import static com.swirlds.platform.crypto.CryptoSetup.initNodeSecurity;
+import static com.swirlds.platform.gui.internal.BrowserWindowManager.getPlatforms;
+import static com.swirlds.platform.state.signed.StartupStateUtils.getInitialState;
+import static com.swirlds.platform.util.BootstrapUtils.checkNodesToRun;
+import static com.swirlds.platform.util.BootstrapUtils.detectSoftwareUpgrade;
+
+import com.swirlds.base.time.Time;
+import com.swirlds.common.config.BasicConfig;
+import com.swirlds.common.config.ConfigUtils;
+import com.swirlds.common.config.StateConfig;
+import com.swirlds.common.context.DefaultPlatformContext;
+import com.swirlds.common.context.PlatformContext;
+import com.swirlds.common.io.utility.RecycleBinImpl;
+import com.swirlds.common.system.NodeId;
+import com.swirlds.common.system.SoftwareVersion;
+import com.swirlds.common.system.SwirldState;
+import com.swirlds.common.system.address.AddressBook;
+import com.swirlds.config.api.Configuration;
+import com.swirlds.config.api.ConfigurationBuilder;
+import com.swirlds.platform.config.internal.PlatformConfigUtils;
+import com.swirlds.platform.config.legacy.LegacyConfigProperties;
+import com.swirlds.platform.config.legacy.LegacyConfigPropertiesLoader;
+import com.swirlds.platform.internal.SignedStateLoadingException;
+import com.swirlds.platform.recovery.EmergencyRecoveryManager;
+import com.swirlds.platform.state.State;
+import com.swirlds.platform.state.address.AddressBookInitializer;
+import com.swirlds.platform.state.signed.ReservedSignedState;
+import com.swirlds.platform.system.Shutdown;
+import com.swirlds.platform.util.BootstrapUtils;
+import com.swirlds.platform.util.MetricsDocUtils;
+import edu.umd.cs.findbugs.annotations.NonNull;
+import edu.umd.cs.findbugs.annotations.Nullable;
+import java.nio.file.Path;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+import java.util.Set;
+import java.util.function.Supplier;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+/**
+ * Builds a {@link SwirldsPlatform} instance.
+ */
+public final class PlatformBuilder {
+
+    private static final Logger logger = LogManager.getLogger(PlatformBuilder.class);
+
+    private final String appName;
+    private final SoftwareVersion softwareVersion;
+    private final Supplier<SwirldState> genesisStateBuilder;
+    private final NodeId selfId;
+    private final String swirldName;
+
+    private ConfigurationBuilder configurationBuilder;
+
+    private static final String SWIRLDS_PACKAGE = "com.swirlds";
+
+    /**
+     * The path to config.txt.
+     */
+    private Path configPath = getAbsolutePath("config.txt");
```

Comment:
Here you do it without `"./"` so we should have the path written in same style everywhere. I prefer this one.